kagami/webui/src/page/IntroductionPage/ListSection/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from 'react';
+import { getResource, ResourceInfo } from '../../../api/resourceInfo';
+import styles from './styles.module.scss';
+
+const ListSection: React.FC = () => {
+  const [resource, setResource] = useState<ResourceInfo[]>([]); // 原始数据
+  const [filteredResource, setFilteredResource] = useState<ResourceInfo[]>([]); // 过滤后的数据
+  const [searchQuery, setSearchQuery] = useState(''); // 搜索框内容
+
+  // 获取资源数据
+  useEffect(() => {
+    const fetchResource = async () => {
+      try {
+        const data = await getResource();
+        setResource(data);
+        setFilteredResource(data); // 默认显示全部
+      } catch (error) {
+        console.error('加载资源失败:', error);
+      }
+    };
+    fetchResource();
```

Comment:
Please also check [@typescript-eslint/no-floating-promises](https://typescript-eslint.io/rules/no-floating-promises), consider `await` it or `void` it